relay-cabi/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "relay-cabi"
-version = "0.8.56"
+version = "0.8.57"
```

Comment:
The version change seems to be done automatically by a bot.

Reply:
Ah ok, so then I'll revert this part?

Reply:
Correct, the release is done with a github action.